### PR TITLE
add sparkle

### LINKED ISSUE #335: add sparkle when looking a new medal
### Describe the feature

add sparkle when looking a new medal

### Why is this feature needed?

for a better user experience

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #335

### CODE REVIEW

**Comment by florian-sabonchi** [comment]
looks good! however there is still a bug with the width of the anmimatiom in landscape mode, do we want to support this=.

**Comment by DasProffi** [comment]
> looks good! however there is still a bug with the width of the anmimatiom in landscape mode, do we want to support this?

As our entire application, doesn't actively support it, I don't think it matters yet, but we should definitely keep it in mind and discuss the landscape mode for impulse

File: apps/impulse/lib/components/star.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    final double midX = width / 2;
+    final double midY = height / 2;
+    final double radius = width / 2;
+    final double innerRadius = radius / 2.5;
+
+    final Path path = Path();
+
+    path.moveTo(midX, 0);
+    for (int i = 0; i < 9; i++) {
+      double angle = i * 2 * 3.141592653589793238 / 8;
```
Comment (florian-sabonchi) [inline apps/impulse/lib/components/star.dart:21]:
```suggestion
      double angle = i * 2 * 3.141592653589793238 / 8;
```

**Comment by use-to** [comment]
This is how helpwave sparkle persists in our hearts 👍🏻

**Pull request merged**